openapi_python_client/resolver/reference.py

```diff
@@ -0,0 +1,51 @@
+import urllib.parse
+
+from .pointer import Pointer
+
+
+class Reference:
```

Comment:
I made some refactoring on the `Reference` object, that as has been quickly integrated to the `ResolvedSchema`, but not tested, there is maybe some thing to fix there